tmatrix.6

```diff
@@ -58,7 +58,7 @@ The minimal length value is \fB1\fR.
 Default: \fI1,30\fR.
 .TP 4
 \-r <min>,<max>
-Set the range for the update rate of the Matrix characters.
+Set the range for the character update rate.
```

Comment:
This is a good start but the whole man paged should be changed to reflect the changes in the help menu.

Reply:
I updated the parameter names. Probably can work on different things later.